packages/wonder-blocks-modal/components/one-column-modal.js

```diff
@@ -105,21 +111,20 @@ class OneColumnModal extends React.Component<Props> {
                     (mediaSize) => mediaSize !== "small" && styles.largeDialog,
                 ]}
             >
-                <ContentWrapper {...this.props} />
+                <WrappedContentWrapper {...this.props} />
             </ModalDialog>
         );
     }
 }
 
-export default MediaLayoutWrapper(OneColumnModal);
-
 const styles = StyleSheet.create({
     dialog: {
         background: Color.white,
     },
 
     largeDialog: {
         width: "64.65%",
+        maxWidth: 960,
```

Comment:
Shouldn't this be `maxWidth: 662`?  Otherwise the lines may be two long.

I think not setting the height for `OneColumnModal` is a good call, but we should probably have a maxHeight just in case there's too much content for whatever reason, maybe `90%`.

Reply:
🛑 this is blocking.

Reply:
Done! Updated to 662 and added `maxHeight: "90%"`